packages/apollo-storybook-vue/example/stories/index.js

```diff
@@ -0,0 +1,20 @@
+import Vue from 'vue';
+import { storiesOf } from '@storybook/vue';
+import gql from 'graphql-tag';
+import HelloWorld from './HelloWorld.vue';
+
+const sampleQuery = gql`
+  query hello {
+    helloWorld
+  }
+`;
+
+storiesOf('Apollo Client', module).add('Hello World Test', () => {
+  return {
+    components: { HelloWorld },
+    template: '<helloWorld  />',
```

Comment:
You'll want to pass the `helloWorld` string through here: `:helloWorld="helloWorld"`.